backend/compact-connect/lambdas/license-data/event_batch_writer.py

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
The newlines are just a habit, since most modules like this include an import or two at the top there. I'm happy to remove. Is the class doc string unclear? I'm happy to rewrite it, I'm just not sure where the description is falling short:
```   """
    Utility class to batch event bridge event puts for better efficiency with the API
    """```

Reply:
I think I understood this more as I did the code review. Regardless, maybe something minor like this? It's just unclear to me what `client` is supposed to be a client for, if that makes sense. Perhaps we could add a type?

```
"""
Utility class to batch events for better efficiency with the AWS ____ API.
"""
```